clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -444,44 +449,66 @@ protected void handleTagCommand() {
       }
     } else if (CommandActions.CREATE.equals(command)) {
       String comment = line.getOptionValue(GravitinoOptions.COMMENT);
-      new CreateTag(url, ignore, metalake, tag, comment).handle();
+      new CreateTag(url, ignore, metalake, tags, comment).handle();
     } else if (CommandActions.DELETE.equals(command)) {
       boolean force = line.hasOption(GravitinoOptions.FORCE);
-      new DeleteTag(url, ignore, force, metalake, tag).handle();
+      new DeleteTag(url, ignore, force, metalake, tags).handle();
     } else if (CommandActions.SET.equals(command)) {
       String property = line.getOptionValue(GravitinoOptions.PROPERTY);
       String value = line.getOptionValue(GravitinoOptions.VALUE);
-
       if (name == null && property != null && value != null) {
-        new SetTagProperty(url, ignore, metalake, tag, property, value).handle();
+        if (tags.length > 1) {
```

Comment:
The check of tags length for SET can be optimized.
I.e. this situation can be checked in the util method?

Reply:
fixed